src/app/Main/Body/QueryPage/ExportDialog/ExportDialog.component.tsx

```diff
@@ -32,6 +33,10 @@ export function ExportDialogComponent(props: {
 
     const handleExport = (): Promise<void> => props.onClickExport();
 
+    const buttonLabel: JSX.Element = ExportButtonLabelComponent(
+        props.exportDialogIsOpen
```

Comment:
I think this could be just `ExportButtonLabelComponent(true)`. Or is there a situation where the dialog is not opened and the button is visible?

Reply:
Done.